### ISSUE TITLE
Cannot connect to Server

### ISSUE THREAD

**Issue by bogatoons** [inspireui/support#4081]
**_Step 1 (require): describe detail issues & screenshots_**
+ Detail Issues: 
+ Product & version:
+ Flutter (or React Native) version: 
+ Testing Device/Simulator: 
+ Screenshot issues (drag the file to attach here): 
![IMG_2661](https://user-images.githubusercontent.com/67900863/86580673-8d878c80-bf6e-11ea-8548-ff55df7c4aaf.jpeg)
I tried to create a new vendor product but got the error " Can not Connect to server"

**_Step 2 (require): submit proof of purchasing the license on http://verify.inspireui.com

_**

**_Important Note:_**
- We will close the ticket if missing the proof of purchase on step 2. The screenshot purchase attaches here will be invalid.
- Kindly create only *One Ticket* & includes all the issues, that would help us focus to resolve it better. 
- If your all ticket was resolved & closed, but want to create a new ticket, just do simple step by linking to the previous verified ID (you don't need to submit the form again), for example, #Ticket-ID

Thank you so much for your time 😊

**Comment by luyx2412** [comment]
Hi @bogatoons Can you describe specific about the issue? I just tested your config and everything worked normal.

Thanks

**Comment by bogatoons** [comment]
I get the Error Cannot connect to Server when a vendor wants to post a new product 

Sent from Yahoo Mail on Android 
 
  On Mon, 6 Jul 2020 at 14:03, luyx2412<notifications@github.com> wrote:   


Hi @bogatoons Can you describe specific about the issue? I just tested your config and everything worked normal.

Thanks

**Comment by bogatoons** [comment]
This is the Error I get when adding new product as a vendor 

Sent from Yahoo Mail on Android 
 
  On Mon, 6 Jul 2020 at 14:03, luyx2412<notifications@github.com> wrote:   


Hi @bogatoons Can you describe specific about the issue? I just tested your config and everything worked normal.

Thanks

**Comment by bogatoons** [comment]
Please help fix this issue

**Comment by bogatoons** [comment]
The delete, edit and vendor product upload doesnt work. What could be the issue?

**Comment by luyx2412** [comment]
Hi @bogatoons Can you share me your project via inspireui.team@gmail.com let me investigate and resend you soon.

Thanks

**Comment by bogatoons** [comment]
Okay thanks I will

Sent from Yahoo Mail on Android 
 
  On Wed, 8 Jul 2020 at 14:34, luyx2412<notifications@github.com> wrote:   


Hi @bogatoons Can you share me your project via inspireui.team@gmail.com let me investigate and resend you soon.

Thanks

**Comment by bogatoons** [comment]
Hi @luyx2412 I've shared my project to the given mail. waiting on you for the reply. thanks

**Comment by luyx2412** [comment]
Hi @bogatoons Sorry i dont see your email, please add title email is `4081`. 

Thanks

**Comment by bogatoons** [comment]
okay , have sent it

**Comment by bogatoons** [comment]
have you seen it yet?

**Comment by bogatoons** [comment]
i have issues with WCFM marketplace too. vendors don't show . products of vendors dont show too.
![Screenshot_1594440815](https://user-images.githubusercontent.com/67900863/87220897-568eed80-c357-11ea-9966-a139ffe1ffc6.png)
![Screenshot_1594440803](https://user-images.githubusercontent.com/67900863/87220899-58f14780-c357-11ea-84c0-c49a67429fca.png)
![Screenshot_1594440499](https://user-images.githubusercontent.com/67900863/87220901-5989de00-c357-11ea-9eb5-6d0ed6dfc147.png)

**Comment by bogatoons** [comment]
its been 5 days and not even the first issue has been solved . Please help me out on my issues . They are very urgent.Thanks @luyx2412

**Comment by luyx2412** [comment]
Hi @bogatoons Sorry about reply you late. Please check your config in wordpress dashboard, when get vendors it showed 404 url.
![image](https://user-images.githubusercontent.com/8909619/87226401-af916c80-c3bd-11ea-94b2-6a4f3626d6c1.png)

Thanks

**Comment by bogatoons** [comment]
hi @luyx2412 i am not using Dokan anymore thats why its showing 404. am using WCFM marketplace

**Comment by bogatoons** [comment]
https://mnmcart.com/wp-json/wcfmmp/v1/store-vendors        @luyx2412

**Comment by bogatoons** [comment]
hello @luyx2412

**Comment by luyx2412** [comment]
Hi @bogatoons In `src/components/VendorInfo/index.js` please replace this file 
[index.js.zip](https://github.com/inspireui/support/files/4913163/index.js.zip)
And `src/common/Tools.js` with 
[Tools.js.zip](https://github.com/inspireui/support/files/4913165/Tools.js.zip) will fix your issue. Because the new version changed field.

Thanks

**Comment by bogatoons** [comment]
okay thanks. can you please check the CustomAPI.js file   with the changes u gave me if i dont make changes in the CustomAPI.js it doesnt work. buh after changing the Dokan/v1 to wcfmmp/V1 it works . but i still dont get the vendor banner and profile picture. @luyx2412

**Comment by bogatoons** [comment]
and the first issue isn't still solved. I get the Error " Can not connect to server " when creating a new product.

**Comment by bogatoons** [comment]
i have solved the issue with the vendor banner. how do i display products by a vendor in the VendorInfo like in the Demo app?
![InkedPHOTO-2020-07-13-16-25-05_LI](https://user-images.githubusercontent.com/67900863/87329333-4da04680-c526-11ea-8dd9-0fb4032a2a0c.jpg)
![InkedPHOTO-2020-07-13-16-25-05 (1)_LI](https://user-images.githubusercontent.com/67900863/87329352-55f88180-c526-11ea-9760-eb3291e1bb3d.jpg)

**Comment by bogatoons** [comment]
hello @luyx2412

**Comment by bogatoons** [comment]
please check the wcfm vendors in the code.the vendor avatar doesnt show, the vendor products doesnt show. most vendor features in the wcfm doesnt show. please help fix these.Thanks @luyx2412 
![Screenshot_1594763646](https://user-images.githubusercontent.com/67900863/87481181-60964200-c61e-11ea-877b-fc7c731180c2.png)
![InkedScreenshot_1594762735_LI](https://user-images.githubusercontent.com/67900863/87481252-83c0f180-c61e-11ea-9b86-b300a938860a.jpg)
![InkedScreenshot_1594762596_LI](https://user-images.githubusercontent.com/67900863/87481258-858ab500-c61e-11ea-954b-4df57b965ef4.jpg)

**Comment by bogatoons** [comment]
please check this too. the vendor chat displays undefined
![Screenshot_1594771005](https://user-images.githubusercontent.com/67900863/87487696-e2da3280-c62d-11ea-95cd-fa167e24be80.png)

**Comment by bogatoons** [comment]
the review also doesnt work after clicking the Submit review button.
![Screenshot_1594771138](https://user-images.githubusercontent.com/67900863/87487778-1b7a0c00-c62e-11ea-9f2c-8b23dc6052a1.png)

**Comment by bogatoons** [comment]
![Screenshot_1594864116](https://user-images.githubusercontent.com/67900863/87617362-dbd32300-c706-11ea-8032-f86eb6411607.png)
 
i get this error too when i try to use the chat feature. please kindly help solve all my issues.thanks @luyx2412

**Comment by luyx2412** [comment]
Hi @bogatoons Can you send me your latest source code. I will fix and reply you soon.

Thanks

**Comment by bogatoons** [comment]
okay thanks. i just sent it

**Comment by minhcasi** [comment]
Please note that we will not support answering or support technical issues for the regular license as per support policies - https://inspireui.com/support-policies/ - but we are trying to best to give extra miles with the supporting. 

We understand that there are some issues when integrating with a new website due to the user-data, kindly upgrade your license to Extend License to continue with the technical support, or you could also process refund it not satisfied or could not use the product - thank you for your time!

**Comment by bogatoons** [comment]
hi @minhcasi , the app is not working as in the Demo. i bought it because i used the Demo and it was good.i am asking for support because the app is not working as to the one provided in the demo. And the issue is not from my side of the integration. some of the codes don't support WCFM which you specified it does support. so please kindly help me solve them.Thanks for the time

**Comment by bogatoons** [comment]
hi @luyx2412 i've sent the project. please have you seen it?

**Comment by bogatoons** [comment]
@minhcasi @luyx2412 is there any problem with the mstore-api. my app is not connecting again

**Comment by bogatoons** [comment]
https://github.com/inspireui/support/issues/4081#issuecomment-658469738   can you please help solve the issue with the chatscreen here. i will try solve the other issues myself.

**Comment by minhcasi** [comment]
@bogatoons - the undefined issue could cause by missing data user data that it could not pick the email info. Please help to check if you have add the Vendor detail info. There are also limit of the WCFM API that could cause the issue - please help to try your best with the checking

**Comment by bogatoons** [comment]
hi @minhcasi thanks for the reply. this is the vendor detail info i got in Postman.
![Screenshot (171)](https://user-images.githubusercontent.com/67900863/87869803-113f7100-c992-11ea-940e-0ee5ee0a2643.png)

**Comment by bogatoons** [comment]
hi @minhcasi i have solved the issue with the chat system . now i can see the messages in firebase but i cant see them in the chat list
![WhatsApp Image 2020-07-19 at 16 11 09](https://user-images.githubusercontent.com/67900863/87879545-ffce8700-c9da-11ea-81d2-3dad20b08728.jpeg)

**Comment by bogatoons** [comment]
hi @minhcasi @luyx2412 i am not getting data in my app again. i updated the mstore api plugin and the purchase code doesnt verify. how do i solve this?

**Comment by bogatoons** [comment]
hi @luyx2412 @minhcasi i have solved the issue with the chatlist screen. i can see the list of chats but the other issue is when i click it , it doesnt open the chat .what could be the issue?


![Screenshot_1595453796](https://user-images.githubusercontent.com/67900863/88231964-92ce2200-cc64-11ea-8ee7-4c85ac65eb38.png)

**Comment by bogatoons** [comment]
![Screenshot_1595518979](https://user-images.githubusercontent.com/67900863/88307303-4c77d200-ccfb-11ea-80a2-c1e3d7d520b4.png)

@luyx2412 @minhcasi after clicking on a chat in the chatlist it doesnt display any messages. i get a blank screen like in the image  . really need your help on this. what could be the problem?
